### PR TITLE
Responsive layout

### PR DESCRIPTION
**Pull request by Takeno**
Some changes for a better experience on mobile devices

### CODE REVIEW

File: partials/socials.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div class="socials">
+<div class="socials row">
   {{#each socials}}
-    <a href="{{this}}" target="_blank"><i class="fa fa-{{@key}}"></i></a>
+    <a class="col-xs-4 col-sm-2 col-md-2" href="{{this}}" target="_blank"><i class="fa fa-{{@key}}"></i></a>
```
Comment (LucaLanziani) [inline partials/socials.hbs:3]: @Takeno I was trying to avoid this in purpose because if we add new social icons it will break the layout

Comment (Takeno) [inline partials/socials.hbs:3]: @LucaLanziani Socials aren't daily posts. I think we can fix layout when we should have to put new icons.
Now is a row of 6 icons on desktop, two row of 3 icons on mobile. A new icon can be change the layout into 1x4 + 1x3 on both for example.

Let me know if keep the original layout implementing custom grid.

/cc @gabrielem

File: layouts/base.hbs [hunk 2]
```diff
@@ -12,9 +12,8 @@
     <div class="container content">
       {{> body }}
     </div>
-    <div class="container-fluid footer">
```
Comment (LucaLanziani) [inline layouts/base.hbs:15]: This was intentional, I did like the footer to be `container-fluid`

File: partials/footer.hbs [hunk 3]
```diff
@@ -1,3 +1,3 @@
-<footer>
+<footer class="container footer  col-xs-10 col-xs-offset-1">
```
Comment (LucaLanziani) [inline partials/footer.hbs:1]: Here the point was to leave the page decide how to render the footer. Especially when and if we are going to add a `blogpost` template

File: partials/head.hbs [hunk 4]
```diff
@@ -1,7 +1,8 @@
 <head>
   <meta charset="utf-8">
-  <link rel="stylesheet" href="/css/style.css" media="screen" title="no title" charset="utf-8">
+  <link rel="stylesheet" href="/css/style.css" media="screen" title="no title" charset="utf-8" />
   <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
```
Comment (LucaLanziani) [inline partials/head.hbs:5]: I <3 this :D

File: layouts/base.hbs (hunk 2 above)
Comment (Takeno) [inline layouts/base.hbs:15]: But it wasn't consistent between Home Page and Internal Page.

File: partials/footer.hbs (hunk 3 above)
Comment (Takeno) [inline partials/footer.hbs:1]: I don't remember sites with different footers between pages. I think the general layout must be consistent. Aren't you?

File: layouts/base.hbs (hunk 2 above)
Comment (LucaLanziani) [inline layouts/base.hbs:15]: The Home Page didn't have a footer at all but just a list of socials, I'm not saying is not ok to add the footer to the homepage but it is a different design ;).

File: partials/footer.hbs (hunk 3 above)
Comment (LucaLanziani) [inline partials/footer.hbs:1]: What does happen if a page has infinite scrolling for instance? 
Of course it is possible for different pages have different footer, if they are different section of a site the header and the footer, for instance, can render differently: different color, different size, etc etc

**Comment by LucaLanziani** [comment]
BTW, I'm going to merge this and we can keep discussing about details on slack or in person ;)

**Pull request merged**